include/Flow.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Content: reads water flow sensor and creates packets, returns string and package
+ * Author : Ryan Leung
+ * Team   : Sustaingineering UBC
+ */
+ 
+#pragma once
+#include <Arduino.h>
+#include "LoRaTransceiver.h"
+#include "farmSensor.h"
+ 
+enum sensorType { digital, analog, counter };
```

Comment:
Not needed here since it is defined in farmsensor.h which is included in this file too